CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to PlexMediaPlayer
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to PMP. 
+
+#### 1 - Read the Plex CLA
+
+You will need to read the CLA which can be found in our [wiki](https://github.com/plexinc/plex-media-player/wiki/Contributor-License-Agreement#individual-agreement) 
+
+You commit message should include mention to it in the last line ie :
+
+> Plex-CLA-1.0-signed-off-by: Your Name  <your.name@email.com>
+
+#### 2 - General Guidelines
+
+There are the few things that you should check before submitting a PR.
+
+ - Commit messages should be clear enough and start with a topic that they are addressing. they typically should look like : *topic : description*. ie *player : what it fixes* ...
```

Comment:
Drop `enough`. Change `start with a topic ...` too: `start with the topic being addressed. An example would be;`.